tbx/static_src/sass/components/_icon.scss

```diff
@@ -24,6 +24,11 @@
         width: 100%;
     }
 
+    &--listing-arrow {
```

Comment:
Could you add `icon--listing-arrow` to the list of utility classes designed for re-use in the docs? https://github.com/torchbox/torchbox.com/blob/main/docs/front-end/utility-classes.md, third paragraph.

Reply:
Thanks, have updated